packages/expo-random/android/src/main/java/expo/modules/random/RandomModule.java

```diff
@@ -1,39 +1,40 @@
 package expo.modules.random;
 
-import android.content.Context;
 import android.util.Base64;
 
 import java.security.SecureRandom;
+import java.security.NoSuchAlgorithmException;
 
-import org.unimodules.core.ExportedModule;
-import org.unimodules.core.ModuleRegistry;
-import org.unimodules.core.interfaces.ExpoMethod;
-import org.unimodules.core.Promise;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
 
-public class RandomModule extends ExportedModule {
-
-  SecureRandom mRandom;
```

Comment:
Keep the member variable since the SecureRandom constructor has to loop through security providers.